plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx

```diff
@@ -14,11 +14,12 @@ import { ReadmeDialog } from '../ReadmeDialog/ReadmeDialog';
 
 export type ReadmeCardProps = {
   variant?: InfoCardVariants;
+  maxHeight?: string | number;
 };
 
 export const ReadmeCard = (props: ReadmeCardProps) => {
-  const { variant = 'gridItem' } = props;
-  const maxHeight = variant === 'fullHeight' ? 'none' : '235px';
+  const { variant = 'gridItem', maxHeight: propMaxHeight } = props;
```

Comment:
Super small thing, but why not just call propMaxHeight maxHeight? Clear enough without the "prop"-part.

Reply:
Wouldn't it conflict with `maxHeight` we already have?